examples/kubernetes/dynamic_provisioning_s3/README.md

```diff
@@ -0,0 +1,68 @@
+## Dynamic Provisioning with data repository Example
+This example shows how to create a FSx for Lustre filesystem using persistence volume claim (PVC) with data repository integration.
+
+Amazon FSx for Lustre is deeply integrated with Amazon S3. This integration means that you can seamlessly access the objects stored in your Amazon S3 buckets from applications mounting your Amazon FSx for Lustre file system. Please check [Using Data Repositories](https://docs.aws.amazon.com/fsx/latest/LustreGuide/fsx-data-repositories.html) for details.
+
+### Edit [StorageClass](storageclass.yaml)
+```
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: fsx-sc
+provisioner: fsx.csi.aws.com
+parameters:
+  subnetId: subnet-056da83524edbe641
+  securityGroupIds: sg-086f61ea73388fb6b
+  s3ImportPath: s3://dl-benchmark-dataset
+  s3ExportPath: s3://dl-benchmark-dataset/export
+```
+* subnetId - the subnet ID that the FSx for Lustre filesystem should be created inside.
+* securityGroupIds - a comman separated list of security group IDs that should be attached to the filesystem
+* s3ImportPath(Optional) - S3 data repository you want to copy from S3 to persistent volume
+* s3ExportPath(Optional) - S3 data repository you want to export new or modified files from persistent volume to S3
+
+### Edit [Persistent Volume Claim Spec](claim.yaml)
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: fsx-claim
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: fsx-sc
+  resources:
+    requests:
+      storage: 5Gi
+```
+Update `spec.resource.requests.storage` with the storage capacity to request. The storage capacity value will be rounded up multiplication of 3600GiB.
+
+### Deploy the Application Reading
+Create PVC, storageclass and the pod that consumes the PV:
+```sh
+>> kubectl apply -f examples/kubernetes/dynamic_provisioning/storageclass.yaml
+>> kubectl apply -f examples/kubernetes/dynamic_provisioning/claim.yaml
+>> kubectl apply -f examples/kubernetes/dynamic_provisioning/pod.yaml
+```
+
+### Use Case 1: Acccess S3 files in readonly mode, no write back.
+If you only want to import data and read it with any modification and creation. You can skip `s3ExportPath` parameter in your `storageclass.yaml` configuration.
+
+You can see S3 files have been downloaded in the persistent volume.
+
+```
+kubectl exec -it fsx-app ls /data
+```
+
+### Use case 2: Sync back new created files to s3ExportPath
+
+Pod `fsx-app` create a file `out.txt` in mounted volume, if you want to export this new file.
```

Comment:
should be using `lfs` to sync back? We should also mention `fsx-app` container image should be built with lustre user space tool

Reply:
Update documentation and add these two points.